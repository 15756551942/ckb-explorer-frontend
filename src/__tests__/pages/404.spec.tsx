import React from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'
import NotFoundPage from '../../pages/404'

describe('Active Page', () => {
  it('is defined', () => {
    const app = shallow(<NotFoundPage />)

    expect(app).toBeDefined()
  })

  it('shallow renders with empty data', async () => {
    const wrapper = renderer.create(<NotFoundPage />).toJSON()
    expect(wrapper).toMatchSnapshot()
  })
})
