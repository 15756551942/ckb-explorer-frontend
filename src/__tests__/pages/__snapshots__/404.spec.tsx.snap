// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Active Page shallow renders with empty data 1`] = `
<div
  className="sc-fzqNJr dLfJWR"
>
  <div
    className="sc-fzoyAV iQkuwf container"
  >
    <img
      alt="404"
      className="sc-fzoLag cea-deJ"
      src="pc_404.png"
    />
  </div>
</div>
`;
